specification/cognitiveservices/Language.Conversations/main.tsp

```diff
@@ -482,7 +483,7 @@ model DateTimeResolution extends BaseResolution {
   resolutionKind: resolutionKind.DateTimeResolution;
 
   @doc("An extended ISO 8601 date/time representation as described in (https://github.com/Microsoft/Recognizers-Text/blob/master/Patterns/English/English-DateTime.yaml)")
-  timex: TimeExpression;
+  timex: string;
```

Comment:
@quentinRobinson why aren't these `utcDateTime` then? We already have a type for this. `string` is too broad, and we shouldn't send people to an otherwise unnecessary time format when our SDKs support `utcDateTime` properly.